app.js

```diff
@@ -63,4 +63,268 @@ function fillData() {
   textArea.value = desirializedObject.textArea;
 }
 
-window.addEventListener('load', fillData);
\ No newline at end of file
+window.addEventListener('load', fillData);
+
+// create the card information
+const otherProjects = [
+  {
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    image: './assets/first-card.png',
+    desktopImg: './assets/large.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    name: 'Profesional Art Printing Data',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    image: './assets/rest-card.png',
+    desktopImg: './assets/Img-Placeholder.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
```

Comment:
- [ ] please ensure that you follow the figma design as much as possible. resize and position your image in the desktop popup so it takes the figma design.

your design
![Screenshot (605)_LI](https://user-images.githubusercontent.com/65298048/202849394-c0152ada-baae-49fa-8ad6-5cf12bee076e.jpg)
figma
![Screenshot (599)](https://user-images.githubusercontent.com/65298048/202849407-43016891-c41a-4e07-8173-9f0613b807c7.png)